desktop-src/direct3dhlsl_current/hlsl-built-ins-functions_math_logic.md

```diff
@@ -0,0 +1,3332 @@
+# Math and Logic Operations
+
+### abs
+
+Returns the absolute value of the specified value.
+
+
+
+| ret abs(*x*) |
+|--------------|
+
+
+
+ 
+
+#### Parameters
+
+
+
+| Item                                                   | Description                            |
+|--------------------------------------------------------|----------------------------------------|
+| <span id="x"></span><span id="X"></span>*x*<br/> | \[in\] The specified value.<br/> |
+
+
+
+ 
+
+#### Return Value
+
+The absolute value of the *x* parameter.
+
+#### Type Description
+
+
+
+| Name  | [**Template Type**](dx-graphics-hlsl-intrinsic-functions.md)                                                  | [**Component Type**](dx-graphics-hlsl-intrinsic-functions.md)                 | Size                           |
```

Comment:
These links to `dx-graphics-hlsl-intrinsic-functions.md` aren't right.  Looks like this was replicated throughout this file.

Reply:
Fixed.